config/secrets.yml

```
@@ -13,12 +13,14 @@
 # Shared secrets are available across all environments.
 
 shared:
-  github_client_id:     <%= ENV['GITHUB_CLIENT_ID']     %>
+  github_client_id: <%= ENV['GITHUB_CLIENT_ID'] %>
   github_client_secret: <%= ENV['GITHUB_CLIENT_SECRET'] %>
-  webhook_secret:       <%= ENV['WEBHOOK_SECRET']       %>
-  api_secret:           <%= ENV['API_SECRET']       %>
-  google_client_id:     <%= ENV['GOOGLE_CLIENT_ID']     %>
+  webhook_secret: <%= ENV['WEBHOOK_SECRET'] %>
+  api_secret: <%= ENV['API_SECRET'] %>
+  google_client_id: <%= ENV['GOOGLE_CLIENT_ID'] %>
   google_client_secret: <%= ENV['GOOGLE_CLIENT_SECRET'] %>
+  github_enterprise_enabled: <%= ENV['GITHUB_ENTERPRISE'] %>
```

Related to my earlier `.env` comment above, we might want to change this:

```suggestion
  github_enterprise_enabled: <%= ENV['GITHUB_ENTERPRISE_ENABLED'] == 'true' %>
```

---

I think we should remove this variable (see above)